ui/app/AppLayouts/Onboarding/views/CreatePasswordView.qml

```diff
@@ -39,7 +39,7 @@ Item {
     ColumnLayout {
         spacing: Style.current.bigPadding
         anchors.centerIn: parent
-        width: 416
+        width: 500
```

Comment:
What if we don't specify `width` in here?

Reply:
This is now outdated. resolving